blazor/file-manager/file-operations.md

```diff
@@ -862,6 +862,81 @@ The available options for the sort order are:
 
 ```
 
+## Custom sorting
+
+The FileManager component provides a way to customize the default sort action for a column by defining the `SortComparer` property of FileManagerColumn.The SortComparer data type was the [IComparer](https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.icomparer-1?view=net-7.0&viewFallbackFrom=net-5) interface, so the custom sort comparer class should be implemented in the interface IComparer.
+
+The following example demonstrates how to define custom sort comparer function for the Name column.
```

Comment:
The following example demonstrates how to define custom sort comparer function to achieve natural sorting behavior for the Name column.

Reply:
@keerthanaRajendran -  I have corrected the mentioned line.